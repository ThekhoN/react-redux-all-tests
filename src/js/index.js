import React from 'react';
import {render} from 'react-dom';
import App from './containers/app';
import '../styles-global/style.css';
import store from './store';
import {Provider} from 'react-redux';

const mountNode = document.getElementById('root');
render(
  <Provider store={store}>
    <App />
  </Provider>,
  mountNode);
