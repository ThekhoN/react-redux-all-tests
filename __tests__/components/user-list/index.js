import React from 'react';
import renderer from 'react-test-renderer';
import UserList from '../../../src/js/components/user-list';
import {dummyData} from '../../../src/js/API';

describe('UserList Component renders correctly', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <UserList userData={dummyData[0]} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
