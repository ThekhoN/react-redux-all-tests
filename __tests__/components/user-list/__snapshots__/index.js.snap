// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList Component renders correctly renders correctly 1`] = `
<li
  className={undefined}
>
  <img
    className={undefined}
    src="https://i1.sdlcdn.com/static/img/marketing-mailers/mailer/2016/upload999/male01.png"
  />
</li>
`;
